docs/docs/code/python/README.md

```diff
@@ -246,10 +244,9 @@ To access them, use the `os` module.
 import os
 import requests
 
-def handler(pd: "pipedream"):
-  token = os.environ['TWITTER_API_KEY']
+token = os.environ['TWITTER_API_KEY']
```

Comment:
I would replace the Twitter examples since it's hard for users to use that API now

Reply:
RIP twitter API